### ISSUE TITLE
No option to open from files

### ISSUE THREAD

**Issue by ghost** [SoulForeverInPeace/Boot-PC#1]
Samsung M31 android 12, when I select browse I only get completely useless options such as gallery, notes, etc.

**Comment by SoulForeverInPeace** [comment]
You are right. i use thirdparty file manager. Noticed this just now. Thanks

**Comment by SoulForeverInPeace** [comment]
I spent/wasted a whole day. Looks like for android11+  devices there are lots of restrictions. I will resolve this issue taking time

**Comment by SoulForeverInPeace** [comment]
Btw Text input field support pasting full file path

**Comment by ghost** [comment]
> Btw Text input field support pasting full file path

didn't know that, also i'd suggest an application redesign, it looks really sketchy but I understand your position. take your time.

**Comment by SoulForeverInPeace** [comment]
Thats on the task(plus code too), once everything is organized in a structured way.

**Comment by SoulForeverInPeace** [comment]
>  i'd suggest an application redesign
> 
Thats on the task(plus code too), once everything is organized in a structured way.

**Comment by SoulForeverInPeace** [comment]
Added a a file picker

**Comment by blessingmwiti** [comment]
ayy @SoulForeverInPeace , is this still an issue?

**Comment by SoulForeverInPeace** [comment]
no, however i think the picker  need to be polished.

**Comment by blessingmwiti** [comment]
Aight
